source/fundamentals/builders.txt

```diff
@@ -0,0 +1,67 @@
+========
+Builders
+========
+
+.. default-domain:: mongodb
+
+.. toctree::
+
+   /fundamentals/builders/aggregates
+   /fundamentals/builders/filters
+   /fundamentals/builders/indexes
+   /fundamentals/builders/projections
+   /fundamentals/builders/sort
+   /fundamentals/builders/updates
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+The Java driver provides classes to make it easier to use the CRUD and Aggregation API. The shorthand utility methods
+allow you to build a query more succinctly.
```

Comment:
Could be helpful to add a note that this section includes guides on how to use each of the available builders. As a pattern, we try to identify what information is in the section so that the reader knows whether they are in the right place, or where to go if there's something related but not covered here.